timefold-solver/src/main/java/com/baeldung/timefoldsolver/Employee.java

```diff
@@ -0,0 +1,32 @@
+package com.baeldung.timefoldsolver;
+
+import java.util.Set;
+
+public class Employee {
+
+    private String name;
+    private Set<String> skills;
+
+    public Employee(String name, Set<String> skills) {
+        this.name = name;
+        this.skills = skills;
+    }
+
+    @Override
+    public String toString() {
+        return name;
+    }
+
+    // ************************************************************************
+    // Getters and setters
+    // ************************************************************************
+
```

Comment:
Remove these comment blocks, I don't remember seeing this style of comments in other articles so it would be a bit out of place
```suggestion
```